llvm/test/Instrumentation/MemorySanitizer/Mips32/vararg-mips.ll

```diff
@@ -1,37 +1,20 @@
 ; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
-; RUN: opt < %s -S -passes=msan 2>&1 | FileCheck %s
+; RUN: opt < %s -S -passes=msan -msan-origin-base=0x40000000 -msan-and-mask=0x80000000 2>&1 | FileCheck %s
```

Comment:
why do we need to change base and mask?

Reply:
Because without that, opt will fail with "unsupported architecture" assert, in `MemorySanitizer::initializeModule`

Reply:
This is due to MemorySanitizer not having default `Linux_Mips32_MemoryMapParams`